specs/lib/readAccessorSpec.js

```diff
@@ -0,0 +1,137 @@
+'use strict';
+var Cesium = require('cesium');
+var CesiumMath = Cesium.Math;
+var defined = Cesium.defined;
+var Cartesian2 = Cesium.Cartesian2;
+var Cartesian3 = Cesium.Cartesian3;
+var Cartesian4 = Cesium.Cartesian4;
+
+var fs = require('fs');
+var path = require('path');
+var clone = require('clone');
+var loadGltfUris = require('../../lib/loadGltfUris');
+var addPipelineExtras = require('../../lib/addPipelineExtras');
+var readAccessor = require('../../lib/readAccessor');
+
+var gltfPath = './specs/data/boxTexturedUnoptimized/CesiumTexturedBoxTest.gltf';
+
+describe('bakeAmbientOcclusion', function() {
+    var boxGltf;
+
+    beforeAll(function(done) {
+        fs.readFile(gltfPath, function(err, data) {
+            if (err) {
+                throw err;
+            }
+            else {
+                boxGltf = JSON.parse(data);
+                addPipelineExtras(boxGltf);
+                loadGltfUris(boxGltf, path.dirname(gltfPath), function(err, gltf) {
+                    if (err) {
+                        throw err;
+                    }
+                    done();
+                });
+            }
+        });
+    });
+
+
+    function fitsBounds(min, max, attributes) {
+        // check if the data in values is bounded by min and max precisely
+        var minInValues;
+        var maxInValues;
+        var attributeToArray;
+        var numberValues = 0;
+
+        switch(attributes.type) {
+            case 'number':
+                attributeToArray = function(value) {
```

Comment:
Actually, I think just try to use [findAccessorMinMax](https://github.com/AnalyticalGraphicsInc/gltf-pipeline/blob/master/lib/findAccessorMinMax.js) here to avoid the duplicate code. Then do your comparison in the test